_maps/RandomZLevels/moonoutpost19.dmm

```diff
@@ -5154,22 +5154,10 @@
-	name = "The Xenomorph 'Castes'"
-	},
-/obj/item/paper/fluff/awaymissions/moonoutpost19/research/larva_autopsy{
-	info = "Researcher: <u>Dr. Mark Douglas    </u><br>Date: <u>04/06/2554</u><br><br>Report:<br>After an extremely dangerous, time consuming and costly dissection, we have managed to record and identify several of the organs inside of the first stage of the xenomorph cycle: the larva. This procedure took an extensive amount of time because these creatures have incredibly, almost-comically acidic blood that can melt through almost anything in a few moments. We had to use over a dozen scalpels and retractors to complete the autopsy.<br><br>The larva seems to possess far fewer and quite different organs than that of a human. There is a stomach, with no digestive tract, a heart, which seems to lack any blood-oxygen circulation purpose, and an elongated brain, even though its as dumb as any large cat. It also lacks any liver, kidneys, or other basic organs.<br><br>We can't determine the exact nature of how these creatures grow, nor if they gain organs as they become adults. The larger breeds of xenomorph are too dangerous to kill and capture to give us an accurate answer to these questions. All that we can conclude is that being able to function with so little and yet be so deadly means that these creatures are highly evolved and likely to be extremely durable to various hazards that would otherwise be lethal to humans.";
-	name = "Larva Xenomorph Autopsy Report"
-	},
+/obj/item/paper/fluff/awaymissions/moonoutpost19/research/xeno_hivemind,
```

Comment:
Am I blind or did you not readd these?

Reply:
I added them with the last cleanup I did, I just forgot to clear out the varedits on the map.